metadata/la.json

```diff
@@ -8,6 +8,9 @@
 	"locations": [
```

Comment:
Is it inconsistent to have city names in the `locations` array, but metro codes broken out?